lib/util.js

```diff
@@ -486,6 +494,16 @@ function formatMap(ctx, value, recurseTimes, visibleKeys, keys) {
   return output;
 }
 
+function formatIterator(ctx, value, recurseTimes, visibleKeys, keys) {
+  var output = [];
+  for (let o of value) {
```

Comment:
If you print an iterator, python just prints the type of iterator. May be we should do the same. Otherwise we'll exhaust the iterator as vkurchatkin pointed out.